coalib/misc/Exceptions.py

```diff
@@ -3,6 +3,8 @@
 from coalib.misc import Constants
 from coalib.output.printers.LogPrinter import LogPrinter
 
+from pkg_resources import ContextualVersionConflict, VersionConflict
```

Comment:
I just experimented around with different stuff and I'm now fairly certain that `VersionConflict` is enough: I think `ContextualVersionConflict` is derived from `VersionConflict` and `ContextualVersionConflict` is raised sometimes in the newer versions, but `VersionConflict` is raised always in the older ones.

And when I put just `VersionConflict` with the newer version, it always capturing the exception. Go ahead with it?